app/services/proofing/lexis_nexis/request_signer.rb

```diff
@@ -0,0 +1,47 @@
+module Proofing
+  module LexisNexis
+    class RequestSigner
+      attr_accessor :config, :message_body, :path, :host
+
+      # @param config must respond to base_url, hmac_key_id, and hmac_secret_key
```

Comment:
the YARD way might be like this:
```suggestion
      # @param [#base_url,#hmac_key_id,#hmac_secret_key] config
```